spec/models/filters/criteria/base_spec.rb

```diff
@@ -0,0 +1,269 @@
+        expect(scope).to be_empty
+      end
+    end
+  end
+
+  describe 'separation between projects and reports' do
+    let(:report_role) { create(:role, can_view_assigned_reports: true, can_view_projects: true) }
+    let(:project_role) { create(:role, can_view_projects: true, can_view_assigned_reports: false) }
+    let(:report_user) { create(:acl_user) }
+    let(:project_user) { create(:acl_user) }
+
+    before do
+      # Give both users access to the same organization collection
+      organization_collection = create(:collection)
+      organization_collection.set_viewables({
+                                              organizations: [organization.id],
+                                              projects: [project_1.id, project_2.id, project_3.id],
+                                            })
+
+      # Report user can access all projects for reporting
+      setup_access_control(report_user, report_role, organization_collection)
+
+      # Project user can only access specific projects for client dashboards
+      project_collection = create(:collection)
+      project_collection.set_viewables({ projects: [project_1.id] })
+      setup_access_control(project_user, project_role, project_collection)
+    end
+
+    it 'allows running reports for organization data' do
+      report_filter = ::Filters::FilterBase.new(user_id: report_user.id)
+      report_criteria = described_class.new(input: report_filter, config: config)
+
+      scope = report_criteria.viewable_project_scope
+      project_ids = scope.pluck(:id)
+
+      # Report user can see all projects in the organization for reporting
+      expect(project_ids).to include(project_1.id, project_2.id, project_3.id)
+    end
+
+    it 'restricts client dashboard access to specific projects' do
+      project_filter = ::Filters::FilterBase.new(user_id: project_user.id)
+      project_criteria = described_class.new(input: project_filter, config: config)
+
+      scope = project_criteria.viewable_project_scope
+      project_ids = scope.pluck(:id)
+
+      # Project user cannot access any projects for reporting (no can_view_assigned_reports)
+      expect(project_ids).to be_empty
+    end
+
+    it 'demonstrates client dashboard access vs report access' do
+      # Test report access (uses can_view_assigned_reports)
+      report_scope = GrdaWarehouse::Hud::Project.viewable_by(report_user, permission: :can_view_assigned_reports)
+
+      # Test client dashboard access (uses can_view_projects)
+      dashboard_scope = GrdaWarehouse::Hud::Project.viewable_by(project_user, permission: :can_view_projects)
+
+      # Report user can access all organization projects for reporting
+      expect(report_scope.pluck(:id)).to include(project_1.id, project_2.id, project_3.id)
+
+      # Project user can only access specific projects for client dashboards
+      expect(dashboard_scope.pluck(:id)).to contain_exactly(project_1.id)
+    end
```

Comment:
Again, I think we want to confirm the set returned for `GrdaWarehouse::Hud::Project.viewable_by(report_user, permission: :can_view_projects)`
and
`GrdaWarehouse::Hud::Project.viewable_by(project_user, permission: :can_view_assigned_reports)`
Which, if permissions are set correctly, would both be empty.  Right now, I think `report_user` can see the projects with `can_view_projects`.